docs/usage_guide.rst

```diff
@@ -141,11 +141,12 @@ BTC Derivation gap limit
 
 This is the derivation gap limit that will be used when trying to derive addresses from a bitcoin xpub. For more information check `here <https://blog.blockonomics.co/bitcoin-what-is-this-gap-limit-4f098e52d7e1>`__.
 
-
 Amount Settings
------------------
+^^^^^^^^^^^^^^^
+
+This section lets the user configure how numbers should appear in the UI. Additionally it is possible to modify the rounding mechanism choosing from ``Round up``, ``Round down`` and ``Half even`` (rounds to the nearest value)
```

Comment:
No need for this description here. Please just skip down to the end and add an entry on rounding. As you can see setting from the amount settings has its own mini section with a description.